instrumentation/log/basic_logger.go

```diff
@@ -264,6 +265,13 @@ func (b *basicLogger) Error(message string, params ...*Field) {
 	b.Log("error", message, params...)
 }
 
+func (b *basicLogger) LogFailedExpectation(message string, expected *Field, actual *Field, params ...*Field) {
+	actual.Key = "actual-" + actual.Key
+	expected.Key = "expected-" + expected.Key
+	newParams:= append(params,expected,actual)
```

Comment:
we should also support the case where params is nil (so only the expected/actual is provided)

Reply:
It will work because `params` are always present even if they are not populated with anything.